### PR TITLE
Improve logic statements

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: Source/CardStackController.m [hunk 1]
```diff
@@ -9,7 +9,7 @@
 static const CGFloat CardStackVerticalVelocityLimitWhenMakingCardCurrent = 100.0f;
 static const CGFloat CardStackVerticalVelocityLimitWhenRemovingCard = 100.0f;
 static const CGFloat CardStackTitleBarHeightWhenSearchIsShown = 8.0f;
-static const CGFloat CardStackTitleBarOverlapToAvoidGaps = 1.0f;
+static const CGFloat CardStackOffsetToAvoidAreaBellowTheTitleToBecomeVisible = 1.0f;
```
Comment (attila-at-hyper) [inline Source/CardStackController.m]: I think it's "below" and not "bellow" (= "a deep roaring shout or sound")

File: Source/CardStackController.m [hunk 2]
```diff
@@ -556,15 +555,16 @@ - (void)updateCardLocationsWhileOpening {
 - (CGRect)frameForCardAtIndex:(NSUInteger)index {
     CGRect frame;
 
+    // Note: Cards at the bottom but behind the last card will be positioned 
+    // outside of the visible area, so their title bar won't show up when the 
+    // last card is being moved.
     BOOL isCardAtTheBottomAndCoveredByTheLastCard = (index > self.currentCardIndex && index < self.cards.count - 1);
```
Comment (attila-at-hyper) [inline Source/CardStackController.m]: In this case I think we should rename `isCardAtTheBottomAndCoveredByTheLastCard` to `shouldCardRemainInvisibleEvenIfLastCardIsMoved`

Comment (3lvis) [inline Source/CardStackController.m]: Fixed :)

Comment (3lvis) [inline Source/CardStackController.m]: @attila-at-hyper like this ? https://github.com/hyperoslo/CardStack/commit/286316e11a7cedbbb4184a11cc273642a2ff9ead

**Pull request merged**